### DISCUSSION TITLE
UIC-Nummern als Codes

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#321] [Q&A]
Bei manchen Ländern und/oder Haltestellen gibt es zwar UIC-Stationsnummern, aber keine Abkürzungen.
In meinem aktuellen Fall sind es Montpellier Sud-de-France und Nîmes-Pont du Gard.

D.h., ich muss hier auf die UIC-Stationsnummern als RIL100-Ersatz zurückgreifen.

Was soll ich da nehmen?
1. Nur UIC-Nummer: `87688887`
2. Flagge + UIC-Nummer: `🇫🇷87688887`
3. Flagge + UIC-Nummer, aber ohne Länder-Präfix: `🇫🇷688887`

**Comment by C1710** [comment]
Option 2 finde ich nicht so super, weil es quasi die Länderinformation redundant hinzufügt.
Option 1 hat den Vorteil, dass es einfacher zu Copy/Pasten ist, Option 3 macht es einfacher zu verstehen, um welches Land es geht.

**Reply by C1710** [reply]
Option 1 hat außerdem den Vorteil, dass sie sich sehr leicht unterscheiden lassen von den anderen Code-Varianten. Beginnt ein Code mit einer Zahl, ist es ein UIC-Code.  
Beginnt es mit einer Flagge, ist es ein länderspezifisches Format.  
Beginnt es mit einem Buchstaben, ist es ein RIL100

**Reply by marhei** [reply]
Am liebsten wäre es mir mittlerweile den UIC-Code als separaten Parameter einzufügen.

Grundsätzlich haben auch die zwei Bahnhöfe auf dem Contournement Nîmes-Montepellier richtige Abkürzungen, aber ich hab auf die schnelle auch nichts gefunden.

**Comment by marhei** [comment]
Also: MHV für Sud de France und NPG für Pont du Gard.
